assets/tutorials/multiple_feasible_solutions.jl

```diff
@@ -0,0 +1,240 @@
+# Here a given digit between 0 and 9 is found in the `i`-th row at the `j`-th place:
+@variable(model, 0 <= Digit[i = ROWS, j = PLACES] <= 9, Int)
+# We also need a higher level "term" variable that represents the actual number in each row:
+@variable(model, Term[ROWS] >= 1, Int)
+# The lower bound of 1 is because we want to get back non-zero solutions.
+
+# Defining the objective function,
+# since this is a feasibility problem, it can just be set to 0:
+@objective(model, Max, 0)
+# @objective(model, Max, Digit[1,0]) #src
+
+# Make sure the leading digit of each row is not zero:
+@constraint(model, NonZeroLead[i in ROWS], Digit[i, (number_of_digits-1)] >= 1)
+
+# Define the terms from the digits:
+@constraint(
+    model,
+    TermDef[i in ROWS],
+    Term[i] == sum((10^j) * Digit[i, j] for j in PLACES)
+)
+
+# The sum of the first three terms equals the last term:
+@constraint(
+    model,
+    SumHolds,
+    Term[number_of_digits] == sum(Term[i] for i in 1:(number_of_digits-1))
+)
+
+# The square is symmetric, that is, the sum should work either row-wise or column-wise:
+@constraint(
+    model,
+    Symmetry[i in ROWS, j in PLACES; i + j <= (number_of_digits - 1)],
+    Digit[i, j] == Digit[number_of_digits-j, number_of_digits-i]
+)
+
+# We also want to make sure we use each digit exactly once on the diagonal or upper triangular region.
+# The following set, along with the collection of binary variables and constraints, ensures this property.
+COMPS = [
+    (i, j, k, m) for i in ROWS for j in PLACES for k in ROWS for
+    m in PLACES if (
+        i + j <= number_of_digits &&
+        k + m <= number_of_digits &&
+        (i, j) < (k, m)
+    )
+]
+
+@variable(model, BinDiffs[COMPS], Bin)
+
+@constraint(
+    model,
+    AllDiffLo[(i, j, k, m) in COMPS],
+    Digit[i, j] <= Digit[k, m] - 1 + 50 * BinDiffs[(i, j, k, m)]
+)
+
+@constraint(
+    model,
+    AllDiffHi[(i, j, k, m) in COMPS],
+    Digit[i, j] >= Digit[k, m] + 1 - 50 * (1 - BinDiffs[(i, j, k, m)])
+)
+
+# Note that the constant 50 is a "big enough" number to make these valid constraints; see 
+# [this paper](https://doi.org/10.1287/ijoc.13.2.96.10515) and 
+# [blog](https://yetanothermathprogrammingconsultant.blogspot.com/2016/05/all-different-and-mixed-integer.html)
+# for more information.
+
+##%% #src
+# We can then call `optimize!` and view the results.
+optimize!(model)
+
+##%% #src
+# Let's check it worked:
+@assert JuMP.termination_status(model) == MOI.OPTIMAL
+@assert JuMP.primal_status(model) == MOI.FEASIBLE_POINT
+
+JuMP.objective_value(model)
+JuMP.value.(Digit,) |> show
+
+# Note the display of `Digit` is reverse of the usual order.
+
+# ### Viewing the Results
+
+# Now that we have results, we can print out the feasible solutions found:
+TermSolutions = Dict()
+for i in 1:result_count(model)
+    TermSolutions[i] = convert.(Int64, round.(value.(Term; result = i).data))
+end
+
+an_optimal_solution = display(TermSolutions[1])
```

Comment:
```suggestion
an_optimal_solution = TermSolutions[1]
#-
```

You can use `#-` to break apart chunks

Reply:
Added.